packages/nextjs/test/serverIntegrations.test.ts

```diff
@@ -0,0 +1,49 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+import { defaultRewriteFrames, getFinalServerIntegrations, IntegrationFunction } from '../src/utils/serverIntegrations';
+
+describe('user integrations without RewriteFrames', () => {
+  test('as an array', () => {
+    const userIntegrations: Integration[] = [];
+    // Should get a single integration: RewriteFrames
+    let finalIntegrations = getFinalServerIntegrations(userIntegrations);
+    expect(Array.isArray(finalIntegrations)).toBeTruthy();
+    finalIntegrations = finalIntegrations as Integration[];
+    expect(finalIntegrations.length).toEqual(1);
+    expect(finalIntegrations[0]).toMatchObject(defaultRewriteFrames);
+  });
+
+  test('as a function', () => {
+    const userIntegrationFnc: IntegrationFunction = (): Integration[] => {
+      return [];
+    };
+    // Should get a single integration: RewriteFrames
+    const integrationWrapper = getFinalServerIntegrations(userIntegrationFnc);
+    expect(typeof integrationWrapper === 'function').toBeTruthy();
+    const finalIntegrations = (integrationWrapper as IntegrationFunction)([]);
+    expect(finalIntegrations.length).toEqual(1);
```

Comment:
```suggestion
    expect(finalIntegrations).toHaveLength(1);
```

https://jestjs.io/docs/expect#tohavelengthnumber